src/components/token/Tokens.js

```diff
@@ -9,6 +9,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
Comment unrelated to the line.

We don't have a linter and we should add it to keep the code consistent and a standard. Just some comments that I've seen from the review and I think could be refactored:

1. single space after `if` statement, e.g. `if (condition)`.
2. `;` in the end of the line.
3. Single space after starting object and before closing it, e.g. `this.setState({ loading: false })`.
4. Single space around operators, e.g. `const nextPage = this.state.page + 1`

We should also add docstring to all methods in this file with a short explanation of what the method does.